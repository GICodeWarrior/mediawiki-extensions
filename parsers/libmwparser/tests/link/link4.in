[[Link]] [[[Link]] [link ] [http://foo bar]  [https://a] [mailto:a] [http://b


