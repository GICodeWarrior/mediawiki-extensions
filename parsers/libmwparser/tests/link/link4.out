BEGIN ARTICLE
    BEGIN PARAGRAPH
        INTERNAL LINK[Link]
        SPACE[ ]
        SPECIAL[[[]
        SPECIAL[[]
        WORD[Link]
        SPECIAL[]]
        SPECIAL[]]
        SPACE[ ]
        SPECIAL[[]
        WORD[link]
        SPACE[ ]
        SPECIAL[]]
        SPACE[ ]
        BEGIN EXTERNAL LINK[http://foo]
            WORD[bar]
        END EXTERNAL LINK
        SPACE[  ]
        EXTERNAL LINK[https://a]
        SPACE[ ]
        EXTERNAL LINK[mailto:a]
        SPACE[ ]
        NEWLINE
        SPECIAL[[]
        WORD[http]
        SPECIAL[:]
        SPECIAL[/]
        SPECIAL[/]
        WORD[b]
    END PARAGRAPH
END ARTICLE
